packages/perseus-editor/src/components/util.ts

```diff
@@ -50,6 +52,8 @@ const DEFAULT_COLOR = "grayH";
 
 export function getDefaultFigureForType(type: "point"): LockedPointType;
 export function getDefaultFigureForType(type: "line"): LockedLineType;
+export function getDefaultFigureForType(type: "circle"): LockedCircleType;
+export function getDefaultFigureForType(type: "vector"): LockedVectorType;
```

Comment:
Just to confirm, you're not adding the Locked vector UI in this PR right (just adding this util type for the future)? I noticed one of the TODOs open for that in a previous file.

Reply:
"Viewing a locked vector" has already been landed! I believe this should have been part of that. I added it here because I was getting TypeScript errors related to it.

I think the TODOs you saw were for "adding a locked vector"